microsoft/botframework-solutions
# [Virtual Assistant Template][TypeScript] Use of 4.3.4 version for botbuilder libraries

## Description

- Use of version `4.3.4` for `botbuilder` libraries
- Update the usage of `botbuilder-skill` in the template
- Update the usage of `botbuilder-skill` in the sample

Related to: #1058 

## Testing Steps
Using the `generator-botbuilder-assistant`

1. Open a terminal wherever you want to create an `assistant`
2. Run `yo botbuilder-assistant`
3. Follow the generator's prompts
4. Check that the generator finished successfully
5. Start the assistant running `npm run start`

## Checklist
\-
